src/oscillators/sa5x_oscillator.c

```diff
@@ -44,10 +45,12 @@
 #define CMD_SET_TAU                   "{set,TauPps0,%d}"
 #define CMD_GET_DISCIPLINING          "{get,Disciplining}"
 #define CMD_SET_DISCIPLINING          "{set,Disciplining,%d}"
+#define CMD_GET_PHASELIMIT            "{get,PhaseLimit}"
+#define CMD_SET_PHASELIMIT            "{set,PhaseLimit,%d}"
 
 #define DISCIPLINING_PHASES 3
 
-static const unsigned int tau_values[DISCIPLINING_PHASES] = {10, 500, 10000};
+static const unsigned int tau_values[DISCIPLINING_PHASES] = {50, 500, 10000};
```

Comment:
Isn't that a revert of #116 @leoleovich ?

Reply:
It’s ok - it seems to be an alternative solution recommended by manufacturer.